CMakeLists.txt

```
@@ -14,7 +14,7 @@ cmake_minimum_required(VERSION 3.4)
 
 # Print warning
 message("#################################################################")
-message("# The CMake build system is currently a beta version.           #")
+message("# The CMake build system does not support all build options.    #")
```

I think the current change sounds a little too negative, maybe something like this is a bit smoother:
```suggestion
message("# The CMake build system may not support advanced build options.    #")
```

---

That's a good idea, thanks!